config.py

```diff
@@ -28,8 +28,8 @@ class BaseConfig(object):
     PROJECT_ROOT = PROJECT_ROOT
     PROJECT_DATABASE_PATH = str(DATA_ROOT)
 
-    SUBMISSIONS_DATABASE_PATH = str(DATA_ROOT / 'submissions')
-    SUBMISSIONS_MIME_TYPE_WHITELIST = [
+    ASSET_GROUP_DATABASE_PATH = str(DATA_ROOT / 'assetGroup')
```

Comment:
Probably best to keep our names in snake-case. Also, for directory and filenames it's probably a good idea to lowercase everything for case insensitive filesystems.